docs/basics/index.md

```diff
@@ -1,8 +1,52 @@
 # Basic Usage
 
+::::{grid}
+
+:::{grid-item-card} Line plot
+:link: line-plot.ipynb
+:img-bottom: ../_static/basics/line-plot.png
+
+:::
+
+:::{grid-item-card} Image plot
+:link: image-plot.ipynb
+:img-bottom: ../_static/basics/image-plot.png
+
+:::
+
+::::{grid-item-card} Slicer plot
+:link: slicer-plot.ipynb
+:img-bottom: ../_static/basics/slicer-plot.png
+
+:::
+
+::::
+
+::::{grid}
+
+:::{grid-item-card} Inspector plot
+:link: inspector-plot.ipynb
+:img-bottom: ../_static/basics/inspector-plot.png
+
+:::
+
+:::{grid-item-card} Super-plot
+:link: super-plot.ipynb
+:img-bottom: ../_static/basics/super-plot.png
+
+:::
+
+:::{grid-item-card} Scatter 3D plot
+:link: scatter3d-plot.ipynb
+:img-bottom: ../_static/basics/scatter3d-plot.png
+
+:::
+
+::::
```

Comment:
Looks like a link to `saving-figures` is missing.

Reply:
Yes, I didn't add it because I didn't have a good image for it.
As it's still in the left hand side menu, I thought it would be ok without it?

Also, if I want to have a card the same size as the other, I have to add two dummy/empty cards next to it.
I could of course just add a single card that spans the whole width?

Reply:
Would look like this
![Screenshot at 2023-11-24 15-40-19](https://github.com/scipp/plopp/assets/39047984/ff355f21-f890-4615-89ef-149697faa021)

Reply:
Would the dummy cards be visible?

Reply:
Actually I can force the number of columns
![Screenshot at 2023-11-24 16-07-00](https://github.com/scipp/plopp/assets/39047984/89afe6c8-b307-4a13-ba26-494c73b82d7b)

Reply:
Ok :+1: